Lib/test/datetimetester.py

```diff
@@ -51,12 +53,33 @@
 # mixed-type comparisons.
 OTHERSTUFF = (10, 34.5, "abc", {}, [], ())
 
-
 # XXX Copied from test_float.
 INF = float("inf")
 NAN = float("nan")
 
 
+def _utcfromtimestamp(klass, ts):
```

Comment:
I wonder why simple timedelta arithmetic will not work here:

```python
def _utcfromtimestamp(klass, ts):
    return klass(1970, 1, 1) + timedelta(seconds=ts)
```

Reply:
@pganssle - I ran test_datetime locally with this variant and all test cases pass.  I am not sure I can push to your branch, but I will try to submit this change somehow to CI.

Reply:
Frankly, given that this is only used in tests, I am not sure how much effort is justified in polishing this, but it is probably good to keep test code simple.